simvue/api/objects/artifact/file.py

```diff
@@ -73,6 +79,19 @@ def new(
             _file_checksum = kwargs.pop("checksum")
         else:
             file_path = pathlib.Path(file_path)
+            if snapshot:
+                _user_config = SimvueConfiguration.fetch()
+
+                _local_staging_dir: pathlib.Path = _user_config.offline.cache.joinpath(
+                    "artifacts"
+                )
+                _local_staging_dir.mkdir(parents=True, exist_ok=True)
+                _local_staging_file = _local_staging_dir.joinpath(
+                    f"{uuid.uuid4()}.file"
```

Comment:
This is a random file name, surely it should have some association with the target?